diff --git a/interpreter\build.gradle b/interpreter\build.gradle
--- a/interpreter\build.gradle
+++ b/interpreter\build.gradle
@@ -12,7 +12,6 @@
 }
 
 dependencies {
-    testCompile group: 'junit', name: 'junit', version: '4.12'
     implementation project (':syntacticanalyzer')
     implementation project (':token')
     implementation project (':astnode')
diff --git a/jsonreaderwriter\build.gradle b/jsonreaderwriter\build.gradle
--- a/jsonreaderwriter\build.gradle
+++ b/jsonreaderwriter\build.gradle
@@ -14,7 +14,6 @@
 dependencies {
     testCompile group: 'junit', name: 'junit', version: '4.12'
     // https://mvnrepository.com/artifact/com.google.code.gson/gson
-    implementation group: 'com.google.code.gson', name: 'gson', version: '2.8.5'
     implementation project (':token')
     implementation project (':astnode')
 }
diff --git a/lexer\build.gradle b/lexer\build.gradle
--- a/lexer\build.gradle
+++ b/lexer\build.gradle
@@ -12,7 +12,6 @@
 }
 
 dependencies {
-    testCompile group: 'junit', name: 'junit', version: '4.12'
     implementation project (':token')
     implementation project (':jsonreaderwriter')
 }
diff --git a/syntacticanalyzer\build.gradle b/syntacticanalyzer\build.gradle
--- a/syntacticanalyzer\build.gradle
+++ b/syntacticanalyzer\build.gradle
@@ -14,7 +14,6 @@
 dependencies {
     implementation 'org.junit.jupiter:junit-jupiter:5.4.2'
     implementation 'org.junit.jupiter:junit-jupiter:5.4.2'
-    testCompile group: 'junit', name: 'junit', version: '4.12'
     implementation project (':token')
     implementation project (':astnode')
     implementation project (':jsonreaderwriter')
